x-pack/legacy/plugins/apm/cypress/package.json

```diff
@@ -5,17 +5,25 @@
   "license": "MIT",
   "scripts": {
     "cypress:open": "cypress open",
-    "cypress:run": "cypress run"
+    "cypress:run": "cypress run --spec **/*.feature",
+    "lint": "tslint --project ./tsconfig.json --fix",
+    "postlint": "npm run tsc",
+    "tsc": "tsc --pretty --noEmit"
```

Comment:
I'd need help in this part, because maybe it's already provided by kibana build

Reply:
I think this can all be removed, typescript should be handled by Kibana scripts, see `src/dev/typescript/projects.ts`